src/components/TableColumnHeader.tsx

```diff
@@ -0,0 +1,30 @@
+import { ArrowUpDown } from 'lucide-react';
+import { Button } from './ui/button';
+import { Column } from '@tanstack/react-table';
+
+type TableColumnHeaderProps<TData, TValue> = {
+  column: Column<TData, TValue>;
+  title: string;
+  buttonClassName?: string;
+  wrapperClassName?: string;
+};
+
+export const TableColumnHeader = <TData, TValue>({
+  column,
+  title,
+  buttonClassName = '',
+  wrapperClassName = '',
```

Comment:
`wrapperClassName` can be just `className`. You pass it to the first element, which does exactly the same thing that you would expect from passing a `className` to do.